run.py

```diff
@@ -0,0 +1,36 @@
+from os import listdir
+import cv2
+from color_detector import ColorDetector
+import numpy as np
+
+blank = cv2.imread("blank.jpg")
```

Comment:
Why do we need this? You can easily call `np.zeros()` if you need a zero'd array https://numpy.org/doc/stable/reference/generated/numpy.zeros.html

Reply:
I have tried that but it wouldn't work, and I couldn't figure out why. I know this isn't the best way to do it but this is the only way that I've been able to show the mask side-by-side with the original image.

Reply:
We should be able to figure this out. What shape were you calling np.zeros with? Is there a dtype used for cv2.imread?

Reply:
So I was actually using np.full() with 255 because I needed a white image, not black. When I wrote this code:
```
blank1 = cv2.imread("blank.jpg")
blank2 = np.full((1080, 1920, 3), 255)
print(np.array_equal(blank1, blank2))
```
The result was that it printed out True.

However, if I simply replaced the line in run.py with the np.full version, it gave me this error:
`OpenCV(4.5.5) /Users/runner/work/opencv-python/opencv-python/opencv/modules/highgui/src/precomp.hpp:155: error: (-215:Assertion failed) src_depth != CV_16F && src_depth != CV_32S in function 'convertToShow'`

And I wasn't sure why or how to fix it.

Reply:
Try `np.full((1080, 1920, 3), 255, dtype=np.uint8)`. This sets the data type of the numpy array entries to uint8 (unsigned integer, 8 bit) which should the 0-255 range that OpenCV is expecting.

Reply:
Thanks Sunny, that works